### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by cxoltero**
I am not too sure about the checkmarks, or if we should just add a reminder.

@thescientist13

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Fixes #
```
Comment (thescientist13) [inline PULL_REQUEST_TEMPLATE.md]: I would consider prompting more about the "solution" than the problem, as ideally the issue will cover that, and in fact, I belive an ISSUE_TEMPLATE would be more appropriate.

So for PRs, consider this
>### Related Issue 
>
>### Changes Made
>
>### PR Checklist
>list of checkboxes []

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Actual Behavior
+
+## Steps to Reproduce the Problem
+
+  1.
+  2.
+  3.
+
+* **Please check if the PR fulfills these requirements**
```
Comment (thescientist13) [inline PULL_REQUEST_TEMPLATE.md]: Maybe a better solution would be to put this into a CONTRIBUTING.md?  Consider this as an example
https://github.com/kenzanlabs/keystone/blob/master/CONTRIBUTING.md

**Comment by cxoltero** [comment]
contributing.md is coming in a second pr with the template files.

**Comment by thescientist13** [comment]
could we just get both in one?  they go together as far as being related A.C. items.

**Comment by thescientist13** [comment]
@cxoltero 
I see a new PR was opened to add the CONTRIBUTING.md, but I would still like to see the changes requested re: PULL_REQUEST_TEMPLATE.md for this PR

**Comment by thescientist13** [comment]
Hey @cxoltero 
Let me know if you have any questions on updating the PR Template, and then we can send this through.

**Pull request closed without merging**